contracts/Pool/Pool.sol

```diff
@@ -31,6 +32,7 @@ contract Pool is Ownable {
     string public name;
     uint public id;
 
+    // todo: Is it really necessary?
```

Comment:
It is only needed for _getManager().retire(). We can add some named payable function and change manager so it will use it.